src/radiotray-ng/main.cpp

```diff
@@ -230,7 +231,21 @@ bool process_command_line_args(int argc, char* argv[])
 
 int main(int argc, char* argv[])
 {
-	if (!process_command_line_args(argc, argv))
+    std::locale::global(std::locale(""));
+
+#if HAVE_GETTEXT
+    const char * i18n_domain = GETTEXT_DOMAIN;
```

Comment:
is this temp variable really necessary?

Reply:
`GETTEXT_DOMAIN` defines in cmake (macros). Its not so simple to view under debugger this value, so I declare `i18n_domain`. I think that its OK.

Reply:
I'd log it at INFO level and do away with the temp variable.

Reply:
> po directory should really be under /src/radiotray-ng

not shure, I use "po file per app chunk" strategy, because `/src/radiotray-ng` contains more than one chunk...